cpp/arcticdb/pipeline/frame_utils.hpp

```diff
@@ -172,6 +169,8 @@ void aggregator_set_data(
                     agg.set_array(col, t);
                 }
             }
+        }  else if constexpr (!is_empty_type(dt)) {
+            static_assert(!sizeof(dt), "Unknown data type");
```

Comment:
This is kind of doing the same thing isn't it? We should do nothing in case of empty type is passed and if `dt` is not empty then we have unhandled type and the code should not compile at all. This is a shorthand for:
```c++
...
else if constexpr (is_empty_type(dt)) {
} else {
            static_assert(!sizeof(dt), "Unknown data type");
}
```
and it's better than throwing since if we have type which is not handled we won't even compile.